d2l-icon.html

```diff
@@ -105,7 +111,7 @@
 				this.async(function() {
 					this.__initialize();
 					this.updateStyles(); // need to re-calc iron-icon variables
-				}, 1);
```

Comment:
There is a big difference between 0 and 1 ms in how Polymer schedules the task: https://www.polymer-project.org/1.0/docs/api/Polymer.Base#method-async